src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -695,25 +734,43 @@ def get_release_namespace(kube_config, kube_context):
 
 def create_cc_resource(client, resource_group_name, cluster_name, cc, no_wait):
     try:
-        return sdk_no_wait(no_wait, client.create, resource_group_name=resource_group_name,
+        return sdk_no_wait(no_wait, client.begin_create, resource_group_name=resource_group_name,
                            cluster_name=cluster_name, connected_cluster=cc)
     except CloudError as e:
         utils.arm_exception_handler(e, consts.Create_ConnectedCluster_Fault_Type, 'Unable to create connected cluster resource')
 
 
+def patch_cc_resource(client, resource_group_name, cluster_name, cc, no_wait):
+    try:
+        return sdk_no_wait(no_wait, client.update, resource_group_name=resource_group_name,
```

Comment:
what is the diff between patch_cc_resource and patch_cc_resource_preview ?.

Reply:
in patch_cc_resource, we use the GA api-version to make patch call. and in patch_cc_resource_preview we use the preview version